pkg/opcfg/config.go

```diff
@@ -29,17 +29,18 @@ import (
 )
 
 const (
-	DefaultZapcoreLevel                   = zapcore.InfoLevel
-	First                                 = 100
-	ThereAfter                            = 100
-	DefaultMaxFileSize                    = 500
-	DefaultMaxFileAge                     = 7
-	DefaultMaxFileRotation                = 3
-	DefaultLevel                          = "info"
-	DefaultDevMode                        = true
-	DefaultVerticaDBConcurrency           = 5
-	DefaultVerticaAutoscalerDBConcurrency = 1
-	DefaultEventTriggerDBConcurrency      = 1
+	DefaultZapcoreLevel                        = zapcore.InfoLevel
+	First                                      = 100
+	ThereAfter                                 = 100
+	DefaultMaxFileSize                         = 500
+	DefaultMaxFileAge                          = 7
+	DefaultMaxFileRotation                     = 3
+	DefaultLevel                               = "info"
+	DefaultDevMode                             = true
+	DefaultVerticaDBConcurrency                = 5
+	DefaultVerticaAutoscalerDBConcurrency      = 1
+	DefaultEventTriggerDBConcurrency           = 1
+	DefaultVerticaRestorePointQueryConcurrency = 1
```

Comment:
We want to be able to control the concurrency through a helm chart. To allow for that we need to update `scripts/template-helm-chart.sh` and `config/default/manager_auth_proxy_patch.yaml` (search for verticadb-concurrency).